components/Form.js

```diff
@@ -39,20 +68,27 @@ export default function Form({
             </option>
           ))}
         </select>
+
         <button type="submit">Submit</button>
         <button type="reset" onClick={handleFormReset}>
           Reset
         </button>
         <button type="button" onClick={handleSurprise}>
           Surprise me
         </button>
-      </form>
-
-      {formResults.length > 0 && <PreviewCard formResults={formResults} />}
+      </StyledForm>
 
       {randomSurprise ? (
         <SpotlightCard randomSurprise={randomSurprise} />
       ) : null}
+      {formResults.length > 0 && <PreviewCard formResults={formResults} />}
+      {formResults === -1 && <h2>no matches...</h2>}
```

Comment:
Looking at the usability: 
When the user clicks the button "surprise card" and afterwards searches for something, the results are only placed under the surprise. This might lead to confusion..

Reply:
Maybe its better to always display on top what the user tried to achieve last (either formResults or randomSurprise depending on the user request